02-peer-protocol.md

```diff
@@ -1032,7 +1270,15 @@ A receiving node:
       - MUST fail the channel.
   - if any `htlc_signature` is not valid for the corresponding HTLC transaction OR non-compliant with LOW-S-standard rule <sup>[LOWS](https://github.com/bitcoin/bitcoin/pull/6769)</sup>:
     - MUST fail the channel.
-  - MUST respond with a `revoke_and_ack` message.
+  - if there is not exactly one `commitsigs` for each splice in progress:
```

Comment:
As far as I can tell, `commitsigs` (quoted) isn't actually defined in this proposal? To what is it referring?

Reply:
It is referring to `commitment_signed`'s `signature` field

Reply:
That should just be:

```suggestion
  - if there is not exactly one `commit_sig` message for each splice in progress:
```

This is much clearer?